.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       
         steps:
             - uses: actions/checkout@v2
-            - run: g++ -Wall character.cpp main.cpp -I character.h -o mosze_01
+            - run: g++ -Wall character.cpp JSONParser.cpp main.cpp -I character.h JSONParser.h -o mosze_01
```

Comment:
Fejfájlnak elvileg itt nem lenne keresnivalója...

Reply:
Itt azért raktam bele a header fájlt is, mivel máshogy nem tudta a g++ lefordítani, arra panaszkodott hogy nem találhatóak a header file-ok.